concourse/steps/component_descriptor.mako

```diff
@@ -170,11 +170,21 @@ base_descriptor_file_v2 = os.path.join(descriptor_out_dir, 'base_component_descr
 with open(base_descriptor_file_v2, 'w') as f:
   f.write(dump_component_descriptor_v2(base_descriptor_v2))
 
+# dump base_archive_v2 as valid component archive and pass it to descriptor script
+base_archive_dir = os.path.join(descriptor_out_dir, 'base_component_archive')
+os.makedirs(base_archive_dir, exist_ok=True)
```

Comment:
nit: as this code runs in a well-defined (containerised..) environment, I would consider the existence of the tgt-dir an error